gatling-engine/src/main/scala/io/snice/gatling/config/Ss7Config.scala

```diff
@@ -0,0 +1,25 @@
+package io.snice.gatling.config
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import io.netty.util.internal.ObjectUtil.checkNotNull
+import io.snice.gatling.ss7.protocol.{M3UAConfig, MtpConfig, SccpConfig, Ss7EngineConfig, TcapConfig}
```

Comment:
```suggestion
import io.snice.gatling.ss7.protocol._
```
nit: in scala we usually import all (not in java)